cppcache/include/geode/CacheableString.hpp

```diff
@@ -308,6 +308,31 @@ inline CacheablePtr createValueArr(const wchar_t* value) {
   return CacheableString::create(value);
 }
 
+template <typename TVALUE>
+inline CacheablePtr createValue(const TVALUE* value) {
+  return CacheableString::create(value);
+}
```

Comment:
Where did this come from? Why would any `TVAL*` cause the creation of a `CacheableString`?

Reply:
This came from `Cacheable.inl` template function `createValue(const TVALUE* value)` which called `createValueArr` which is only defined in `CacheableString.hpp`.  So intent was to unroll that. I didn't move it to `Cacheable.hpp` because then `Cacheable.hpp` and `CacheableString.hpp` would have circular dependency.

Reply:
I am just curious how it handles matching say `int*` since it lacks any conditional traits. I suppose our changes to `std::string` will address this.